test-integration/helpers/window-position.coffee

```diff
@@ -0,0 +1,12 @@
+windowPosition = (test) ->
```

Comment:
Agreed on the clarity of naming vars. But what about naming it `theTest` (or something), or using `const` in ES6 to ensure the var is not renamed? I'm more worried about consistency and I guess just a slight preference for closures :smile: